Webrew.Client/ClientApp/src/app/browse/components/browse/browse.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { BeersService } from 'src/app/services/beer.service';
+
+@Component({
+    selector: 'browse',
+    templateUrl: './browse.component.html',
+    styleUrls: ['./browse.component.scss']
+})
+export class BrowseComponent implements OnInit {
+    constructor(private beersService: BeersService) {}
+
+    beers$;
```

Comment:
Give this a type

Reply:
this is just fake till it gets fully implemented, it needs to handling filtering but I wasn't sure how to do that.